mdata/aggmetric_test.go

```diff
@@ -174,6 +175,10 @@ func BenchmarkAggMetrics1000Metrics1Day(b *testing.B) {
 	}
 
 	metrics := NewAggMetrics(dnstore, chunkSpan, numChunks, chunkMaxStale, metricMaxStale, ttl, 0, aggSettings)
+	go func() {
+		for range chunk.TotalPoints {
```

Comment:
yeah me neither actually.  now that I think about it, we could just replace the channel based approach with atomic.UpdateUint64 or something. then whether we read out the values or not doesn't really matter.  the upcoming met library also uses this approach internally, but i'ld like benchmarks to be functional before then, so i'll patch this up tomorrow I think